server/rating.js

```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import fetch from "node-fetch";
+
+export const displayRating = express.Router();
+
+async function getRatingData(id) {
+  const response = await fetch(
+    "https://plankton-app-xhkom.ondigitalocean.app/api/reviews?filters%5Bmovie%5D=" +
+      id
+  );
+  const reviewData = await response.json();
+  const data = reviewData.data;
+  return data;
+}
+
+export function roundRating(passInData) {
```

Comment:
The `passInData` name does not communicate what it is that is being passed in. Is it an array of numbers? If so, maybe call it `ratingValues` or something?